.github/actions/templates/getWorkflowInput/action.yml

```diff
@@ -1,3 +1,28 @@
+#########################################################
+## 'Get Workflow Input' Composite Action               ##
+#########################################################
+##
+## This pipeline template contains the logic to read pipeline runtime parameters and publih their value as a pipeline output.
+## This is essential in case a pipeline was not triggered via 'workflow_dispatch' as this renders the otherwise available runtime parameters unavailable.
```

Comment:
This is essential in case a module workflow was not triggered via the 'workflow_dispatch' method, so these runtime parameters will not be available to use when the workflow runs.

Reply:
done